packages/devtools_app/lib/src/shared/config_specific/drag_and_drop/drag_and_drop.dart

```diff
@@ -62,7 +62,9 @@ abstract class DragAndDropManager {
   /// newly active [DragAndDrop] widgets accordingly.
   void hitTestAndUpdateActiveId(double x, double y) {
     final hitTestResult = HitTestResult();
-    RendererBinding.instance.hitTest(hitTestResult, Offset(x, y));
+    final viewId =
+        RendererBinding.instance.renderView.flutterView.viewId as int;
```

Comment:
We will deprecate the renderView getter shortly as well for multi view as each view will have its own render tree rooted at its own RenderView. So, there will be multiple RenderViews. To properly support multi view (and multi window) you will have to define what view the (x,v) coordinates you pass in here are in. Typically, if you're in a widget tree you can obtain this information from `View.of(context).viewId` - or if (x,y) are directly taken from a `PointerEvent`, that event also has a `viewId` getter.

I tried to follow the code to see where this (x,y) is ultimately coming from, but github search is failing me as I can't seem to find any instances in this repo that call `dragOver` and pass in (x,y). That would be interesting to know, though, to figure out how you can obtain the viewID that you'd ultimately have to pass down to this method.